ocs_ci/utility/utils.py

```diff
@@ -392,6 +394,9 @@ def run_cmd(cmd, secrets=None, **kwargs):
     log.info(f"Executing command: {masked_cmd}")
     if isinstance(cmd, str):
         cmd = shlex.split(cmd)
+    if not kwargs.get('timeout'):
+        # set a default timeout for any command to timeout within 600 seconds
+        kwargs['timeout'] = 600
```

Comment:
I would email ocs-qe mailing list to ask ppl to identify those commands. Test owners will probably know better in which places the 600 seconds won't be enough. WDYT?

Reply:
I disagree. In some cases the author will likely be aware of that (eg. as Coady noted for the installation), but in most cases I doubt owners will be able to identify most such cases without  rerun and retesting.

Reply:
I would suggest for Vasu to present xml junit files run of all (not just tier1) cases with and without the pr, analyze the differences and only then go to the people asking for help.

Reply:
@mbukatov if Vasu will run all the tiers, compare the results it will take some time and we don't cover all the test cases cause of BZ skips . So I just thought that before even executing and spend this time would be worth to just ask ppl if they remember which commands they are executing in their test cases take more then 600 timeout just in case they remember to give some rough identification. So we can do some alignment int this PR even before the execution all tiers.

Reply:
> compare the results it will take some time and we don't cover all the test cases cause of BZ skips

I understand that you want to optimize the work you need to do to push parts of it to rest of the team. That said, this is the case for every change in the framework which could affect any test. If someone else did a change like that, you would require that the person does all this work. 

That said, do what you consider the best way forward. I just want to point this difference in approach clearly.

Reply:
Anyway, Vasu already sent the email yestarday so he will get maybe some feedback from someone and now it's up to him to do the rest. ;)